.github/workflows/cli.yml

```diff
@@ -0,0 +1,52 @@
+name: CLI testing
+
+on:
+  push:
+    branches:
+      - main
+      - develop
+      - warp-ts
+  pull_request:
+
+jobs:
+  CLI_test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16.x'
+          cache: 'yarn'
+
+      - name: Setup Python
+        uses: actions/setup-python@v2.2.2
+        with:
+          python-version: 3.9 # Cairo was tested with this version
+
+      - name: Install dependencies
+        run: |
+          yarn
+          make compile
+          pip3 install cairo-lang==0.10.3
```

Comment:
Isn't the cairo version supported by warp 10.1? Can you update warp's cairo lang dependency as well?

Reply:
`cairo-format` is buggy in `0.10.3`, It is producing invalid cairo code with no `;` at end for some lines of code.  we can do the dependency update  in separate PR.

Reply:
sure, https://www.notion.so/nethermind/Cairo-dependency-update-0-10-3-2c9c261579054a3fafd8306d8a43f36c